plugins/flutter_aepedge/README.md

```diff
@@ -0,0 +1,183 @@
+
+As part of the initialization code, make sure that you set the SDK wrapper type to `Flutter` before you start the SDK.
+
+Refer to the [Initialization](https://github.com/adobe/aepsdk_flutter#initializing) section of the root README for more information about initializing the SDK.
+
+**Initialization Example**
+
+iOS
+```objc
+// AppDelegate.h
+@import AEPCore;
+@import AEPEdge;
+@import AEPEdgeIdentity;
+...
+@implementation AppDelegate
+
+// AppDelegate.m
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [AEPMobileCore setWrapperType:AEPWrapperTypeFlutter];
+
+     // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+    NSString* ENVIRONMENT_FILE_ID = @"YOUR-APP-ID";
+    
+    const UIApplicationState appState = application.applicationState;
+
+    NSArray *extensionsToRegister = @[AEPMobileEdgeIdentity.class, 
+                                      AEPMobileEdge.class,                                              
+                                      ];
+
+    [AEPMobileCore registerExtensions:extensionsToRegister completion:^{
+    [AEPMobileCore configureWithAppId: ENVIRONMENT_FILE_ID];
+    }];
+    return YES;   
+ } 
+```
+
+Android
+```java
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.identity.Identity;  
+...
+import io.flutter.app.FlutterApplication;
+...
+public class MainApplication extends FlutterApplication {
+  ...
+  // TODO: Set up the preferred Environment File ID from your mobile property configured in Data Collection UI
+  private final String ENVIRONMENT_FILE_ID = "YOUR-APP-ID";
+
+  @Override
+  public void on Create(){
+    super.onCreate();
+    ...
+    MobileCore.setApplication(this);
+    MobileCore.setWrapperType(WrapperType.FLUTTER);
+
+    Edge.registerExtension();
+    Identity.registerExtension();
+    MobileCore.start(new AdobeCallback () {
+        @Override
+        public void call(Object o) {
+          MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
+        }
+   });
+```
+------
+#### Importing the SDK:
+```dart
+import 'package:flutter_aepedge/flutter_aepedge.dart';
+```
+------
+#### Getting edge version:
+ ```dart
+String version = await edge.extensionVersion;
+ ```
+------
+#### sendEvent
+ ```dart
+String version = await edge.extensionVersion;
+ ```
+**Syntax**
+```dart
+static Future<List<EventHandle>> sendEvent(
+    ExperienceEvent experienceEvent,
+  )
+```
+
+**Example**
+```dart
+late List<EventHandle> result;
+Map<String, dynamic> xdmData = {"eventType": "SampleEventType"};
+Map<String, dynamic> data = {"free": "form", "data": "example"};
+
+final ExperienceEvent experienceevent = ExperienceEvent({
+  //xdmData is required for experienceEvent
+  "xdmData": xdmData,
+  "data": data,
+  "datasetIdentifier": "datasetIdExample"
```

Comment:
we can omit this optional parameter here since we include it in the detailed examples below